lua/lualine.lua

```diff
@@ -191,29 +187,11 @@ end
 local function lualine_set_theme()
   if type(M.options.theme) == 'string' then
     M.options.theme = require('lualine.themes.'.. M.options.theme)
-    -- change the theme table in component so their custom
-    -- highlights can reflect theme change
-    local function reset_component_theme(sections)
-      for _, section in pairs(sections)do
-        for _, component in pairs(section) do
-          if type(component) == 'table' then
-            component.theme = M.options.theme
-          end
-        end
-      end
-    end
-    reset_component_theme(M.sections)
-    reset_component_theme(M.inactive_sections)
   end
-  utils.clear_highlights()
```

Comment:
We should still clear the highlights before creating the new ones I think

Reply:
Yeah, I thought this will only run once but sourcing a config file might execute this an additional time. I will revert it